### PR TITLE
payment added

### PR DESCRIPTION
**Pull request by Jahid5550**
please review my payment function

### CODE REVIEW

File: src/store/index.js
Comment (cRED-f) [inline src/store/index.js:1]: you have to add search function in reducer

**Review by cRED-f — CHANGES_REQUESTED** [review]

File: src/store/index.js
Comment (Jahid5550) [reply src/store/index.js:1]: ok

**Comment by Jahid5550** [comment]
done please check

**Review by cRED-f — APPROVED** [review]
ok perfect

**Pull request merged**